src/core/utils/misc.cpp

```diff
@@ -553,7 +553,33 @@ void Utils::Misc::openFolderSelect(const QString& absolutePath)
     if (QFileInfo(path).exists()) {
         // Syntax is: explorer /select, "C:\Folder1\Folder2\file_to_select"
         // Dir separators MUST be win-style slashes
-        QProcess::startDetached("explorer.exe", QStringList() << "/select," << Utils::Fs::toNativePath(absolutePath));
+
+        // QProcess::startDetached() has an obscure bug. If the path has
+        // no spaces and a comma(and maybe other special characters) it doesn't
```

Comment:
It was just the comment, I had the impression that it wasn't explaining the problem.

It's fine like this. go ahead and merge this if it works.